ts/components/Alert/index.tsx

```diff
@@ -114,6 +107,16 @@ const mapVariantStates: Record<NonNullable<Alert["variant"]>, VariantStates> = {
   }
 };
 
+/**
+ *
+ * Displays an alert with various variants (error, warning, info, success) and an optional action.
+ * Currently if the Design System is enabled, the component returns the Alert of the @pagopa/io-app-design-system library
+ * otherwise it returns the legacy component.
+ * @param {Alert} props The component props.
+ *
+ * @deprecated The usage of this component is discouraged as it is being replaced by the ButtonLink of the @pagopa/io-app-design-system library.
```

Comment:
```suggestion
 * @deprecated The usage of this component is discouraged as it is being replaced by the Alert of the @pagopa/io-app-design-system library.
```
did you mean this component?